src/utils/runWithTruffle.js

```diff
@@ -1,12 +1,17 @@
 import Truffle from '../models/truffle/Truffle';
 import Session from '../models/network/Session'
+import Contracts from 'zos-lib/lib/utils/Contracts';
+import _ from 'lodash';
 
-export default async function runWithTruffle(script, network, compile = false) {
+const DEFAULT_TIMEOUT = 10 * 60; // 10 minutes
+
+export default async function runWithTruffle(script, network, { compile = false, timeout = null }) {
```

Comment:
why not using `DEFAULT_TIMEOUT` here instead?

Reply:
I was unsure if `commander` would consistently return `undefined` (and not `null`), thus causing the default value not to be used.

Reply:
I gave it a try and I guess I got an `undefined` but let's be sure about it